automl/beta/src/main/java/com/example/automl/SetEndpoint.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.automl;
+
+import com.google.cloud.automl.v1beta1.AutoMlClient;
+import com.google.cloud.automl.v1beta1.AutoMlSettings;
+import com.google.cloud.automl.v1beta1.Dataset;
+import com.google.cloud.automl.v1beta1.ListDatasetsRequest;
+import com.google.cloud.automl.v1beta1.LocationName;
+
+import java.io.IOException;
+
+class SetEndpoint {
+
+  // Change your endpoint
+  static void setEndpoint(String projectId) throws IOException {
+    // [START automl_set_endpoint]
```

Comment:
Please have the tag include the imports and the request placement.

Reply:
So this tag isn't a full snippet, it is going on a special page where they only want just the code to set the endpoint. 
But I have a full snippet so we can test it.

Reply:
OK fair enough